src/components/Auth/Login/Login.react.js

```diff
@@ -258,6 +260,7 @@ class Login extends Component {
 					<h3><Translate text="Login to SUSI"/></h3>
 					<form onSubmit={this.handleSubmit}>
 						<div>
+						<CommunicationEmail/>
 							<TextField name="email"
```

Comment:
where in the code the margin is given? im not able to locate the line

Reply:
@daminisatya  There is no margin , it doesn't allow responsiveness . The UI looks much better in default values .
U want me to add margin and change it adopt responsiveness ? or this looks fine ?